packages/sdk-cli/src/util/fetchUtil.ts

```diff
@@ -26,6 +20,5 @@ export const assertJSON = (expected = 'application/json') => async (response: Re
 
 export const decodeJSON = <A, O, I>(endpointType: t.Type<A, O, I>) =>
   (response: Response): Promise<A> =>
-    assertOK(response)
```

Comment:
Whoah, this seriously changes the `decodeJSON` contract. Have you audited all uses of `decodeJSON` to make sure that no `assertOK` checks are getting skipped? Maybe `decodeJSON` should be moved into `baseAPI` and use the API-specific `assertOK`.

Reply:
Yes, both uses of decodeJSON call apiFetch then decodeJSON. My thought process is that the `decodeJSON` function shouldn't be responsible for asserting that the response was okay. What if we want to decode a JSON payload for a response that's not a 2xx?

Reply:
Right, but the common case is to use it after an apiFetch which is already making the assertOK check? I'm arguing the assertOK should be done during the apiFetch part not the decodeJSON part.

Reply:
I'm arguing that `assertOK` should not be applied as part of the `apiFetch` since there is no escape hatch for that.